bliss/models/decoder.py

```diff
@@ -484,12 +480,9 @@ def _construct_full_image_from_ptiles(image_ptiles, tile_slen, border_padding):
                 )
 
                 # get canvas
-                canvas[
-                    :,
-                    :,
-                    (i * tile_slen) : (i * tile_slen + canvas_len),
-                    (j * tile_slen) : (j * tile_slen + canvas_len),
-                ] += image_tile_cols
```

Comment:
Black auto-formatting has led to some really unreadable code. I refactored some of it.